src/views/mfa-verify/WindowsHelloErrorMessageView.js

```diff
@@ -18,14 +18,22 @@ define([
 
     var template = '\
       <span class="icon error-24"></span>\
-      <h4><strong>{{i18n code="enroll.windowsHello.error.notWindows" bundle="login"}}</strong></h4>\
+      <h4><strong>{{{message}}}</strong></h4>\
```

Comment:
https://oktawiki.atlassian.net/wiki/display/PM/Windows+10+Hello+Integrated+Experience?preview=/146506847/151487143/03b%20-%20Win%20Hello%202FA%20Error2.png

Here is the mock with the arrows (&gt;) and the bold words, for which I need to use html markup without escaping.